src/test/routes/userProfiles.spec.ts

```diff
@@ -1,4 +1,5 @@
 import { appTest } from "../../main/app.test";
+import { appTestWithAuthroziedAdminWebRoles } from "../../main/app.test-admin-web-roles-authorized";
```

Comment:
Looks like a typo here - guess it should be `appTestWithAuthorizedAdminWebRoles`